broker-daemon/bin/sparkswapd.js

```diff
@@ -5,81 +5,80 @@
 const program = require('caporal')
 
 const BrokerDaemon = require('../')
-const { currencies } = require('../config')
+const config = require('../config')
+const { currencies } = config
+const { version: CLI_VERSION } = require('../../package.json')
 
 // TODO: Change this path to be sparkswapd specific
 const { validations } = require('../../broker-cli/utils')
 
-const { version: CLI_VERSION } = require('../../package.json')
+// LND currently uses ECDSA generated certs, which we use on the broker. We've
+// add ECDSA to the gRPC cipher suites as it is not added by default.
+// If this line is removed, we will see an SSL Handshake Error stating `no shared cipher`
+process.env.GRPC_SSL_CIPHER_SUITES = 'HIGH+ECDSA:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-RSA-AES256-GCM-SHA384'
```

Comment:
did you include this here because this is effectively application code, and not really something that ought to be changed?